### DISCUSSION TITLE
<title>won't let .e delete my account I just want it fucking deleted

### DISCUSSION THREAD

**Discussion by KatEyez1311** [community/community#88451] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm not tech savvy didn't realize what I was signing up for I need customer support to delete my account because I've tried and it won't let me!!!

**Comment by mishmanners** [comment, marked as answer]
Closing this out. Please watch your language in future. If you need specific account support for closing or deleting an account, please open a support ticket: https://docs.github.com/en/support/contacting-github-support

**Reply by TicketWizard** [reply]
Same - How do I close my f'ing account?  I have sent emails - nothing!!

**Reply by lindatyson324-web** [reply]
Sorry my fingers hit that. Thats not my language

**Reply by lindatyson324-web** [reply]
Please help me delete my account

**Comment by ghost** [comment]
I want my account deleted! You are charging me for a service that i did not sign up for or want!

**Comment by Pricer187** [comment]
fuck you watch yourself

**Comment by TicketWizard** [comment]
?
